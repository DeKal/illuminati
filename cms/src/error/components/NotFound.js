import React from 'react'
import PropTypes from 'prop-types'
import Layout from 'error/components/base/Layout'

const NotFound = ({ message }) => {
  return (
    <Layout
      title={'An error 404 occurred'}
      description={
        'The page you are looking for might have been removed had its name changed or is temporarily unavailable.'
      }
      detail={message}
    />
  )
}

export default NotFound

NotFound.propTypes = {
  message: PropTypes.string
}
