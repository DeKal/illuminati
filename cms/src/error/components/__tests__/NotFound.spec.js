import * as React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../NotFound'

describe('NotFound', () => {
  it('should render not found page correctly', function() {
    const wrap = shallow(<NotFound message="Pages not found" />)
    expect(wrap).toMatchSnapshot()
  })
})
