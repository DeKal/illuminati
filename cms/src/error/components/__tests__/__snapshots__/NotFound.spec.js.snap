// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render not found page correctly 1`] = `
<Layout
  description="The page you are looking for might have been removed had its name changed or is temporarily unavailable."
  detail="Pages not found"
  title="An error 404 occurred"
/>
`;
